survey/settings.py

```diff
@@ -138,21 +143,26 @@
 EMAIL_HOST_PASSWORD = 'wvafyygjugpgnhcf'
 DEFAULT_FROM_EMAIL = 'videotakvideo@gmail.com'
 
-
-
 ANYMAIL = {
-    "SENDINBLUE_API_KEY": "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",
+    "SENDINBLUE_API_KEY": "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",
```

Comment:
Similarly, this should be a environment variable in real project. The rule is never push credential keys to Github or any where else.

Reply:
> Similarly, this should be a environment variable in real project. The rule is never push credential keys to Github or any where else.

Forgot to add setting.py to gitignore ) Ok I will create env var